storybookjs/storybook
# Story sorting

Issue: https://github.com/storybooks/storybook/issues/6439

Story sorting now takes place in story_store. This is so that the Object.values returns the correct order and sorting is not needed elsewhere.

## What I did
Added a Param to the global params called storySort which takes a sort function. If none is provided then the default alphabetical sort is used.

## How to test

- Is this testable with Jest or Chromatic screenshots? don't know yet
- Does this need a new example in the kitchen sink apps? Is showcased in react kitchen sink
- Does this need an update to the documentation? yes

If your answer is yes to any of these, please make sure to include it in your PR.
